confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeDescriptor.java

```diff
@@ -175,6 +175,24 @@ private void loadProps() {
               properties.getProperty(
                   "data_region_count", String.valueOf(conf.getDataRegionCount()))));
 
+      String addresses = properties.getProperty("config_node_group_address_list", null);
```

Comment:
Maybe we can abstract these method to iotdb-common so that datanode can use this logic too?

Reply:
Of course, I think we need a common .properties file reading process in iotdb-common. But I think this job isn't urgent currently, we could optimize both IoTDBDescriptor and ConfigNodeDescriptor later.

Reply:
Agree with that